### PR TITLE
Fix some typos in whatsnew

### PR DESCRIPTION
**Pull request by kurtmckee**
I think this is a trivial fix for some typos, so there's currently no bpo number and I haven't modified `Misc/ACKS`.

Let me know if this needs any modification. Thanks for your work on Python!

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).



## CLA Missing

Our records indicate the following people have not signed the CLA:

@kurtmckee

For legal reasons we need all the people listed to sign the CLA before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Doc/whatsnew/3.11.rst [hunk 1]
```diff
@@ -193,9 +193,9 @@ Other CPython Implementation Changes
   support :class:`typing.SupportsComplex` and :class:`typing.SupportsBytes` protocols.
   (Contributed by Mark Dickinson and Dong-hee Na in :issue:`24234`.)
 
-* ``siphash13`` is added as a new internal hashing algorithms. It's has similar security
+* ``siphash13`` is added as a new internal hashing algorithms. It has similar security
   properties as ``siphash24`` but it is slightly faster for long inputs. ``str``, ``bytes``,
-  and some other types now use it as default algorithm for ``hash()``. :pep:`552`
+  and some other types now use it as the default algorithm for ``hash()``. :pep:`552`
```
Comment (merwok) [inline Doc/whatsnew/3.11.rst]: IMO there wasn’t a problem to fix here.

Thanks for catching the other typos!

**Comment by merwok** [comment]
I agree about skipping bug and news, but please sign the CLA.

File: Doc/whatsnew/3.11.rst (hunk 1 above)
Comment (merwok) [reply Doc/whatsnew/3.11.rst]: Yes, please revert this unless you disagree with me!
(Also changed the markup to create a link)

```suggestion
  and some other types now use it as default algorithm for :func:`hash`. :pep:`552`
```

Comment (kurtmckee) [reply Doc/whatsnew/3.11.rst]: I think it needs the definite article "the". The sentence reads less clearly without it. If I have the option to keep the addition then I will, but if it's a blocker for accepting the patch then I'll remove it.

100% agree on converting the `hash` reference to a link. I'll get that done.

Comment (kurtmckee) [inline Doc/whatsnew/3.11.rst]:
```suggestion
  and some other types now use it as the default algorithm for :func:`hash`. :pep:`552`
```

Comment (merwok) [reply Doc/whatsnew/3.11.rst]: But the empty article is a valid article in English, and appropriate here!  «Some types use it as default algorithm» has no mistake.

File: Doc/whatsnew/3.11.rst [hunk 2]
```diff
@@ -193,9 +193,9 @@ Other CPython Implementation Changes
   support :class:`typing.SupportsComplex` and :class:`typing.SupportsBytes` protocols.
   (Contributed by Mark Dickinson and Dong-hee Na in :issue:`24234`.)
 
-* ``siphash13`` is added as a new internal hashing algorithms. It's has similar security
+* ``siphash13`` is added as a new internal hashing algorithms. It has similar security
   properties as ``siphash24`` but it is slightly faster for long inputs. ``str``, ``bytes``,
-  and some other types now use it as default algorithm for ``hash()``. :pep:`552`
+  and some other types now use it as the default algorithm for :func:`hash`. :pep:`552`
```
Comment (kurtmckee) [inline Doc/whatsnew/3.11.rst]:
```suggestion
  and some other types now use it as default algorithm for :func:`hash`. :pep:`552`
```

File: Doc/whatsnew/3.11.rst (hunk 1 above)
Comment (kurtmckee) [reply Doc/whatsnew/3.11.rst]: I've removed the definite article. Thanks for your feedback and work on Python! 👍

**Review by Mariatta — APPROVED** [review]
Thanks.

**Pull request merged**